installer.js

```diff
@@ -85,9 +110,9 @@ function getDataForPlatform() {
         binaryfiles: ['7z','7z.so','7za','7zCon.sfx','7zr','Codecs'],
         sfxmodules: ['7zS2.sfx','7zS2con.sfx','7zSD.sfx'] };
         // Mac version
-        case "darwin": return { url: 'https://raw.githubusercontent.com/rudix-mac/packages/master/' + macosversion + '/', 
-        filename: 'p7zip-9.20.1-1.pkg',
-        extraname: '7z920_extra.7z',
+        case "darwin": return { /* url: 'https://raw.githubusercontent.com/rudix-mac/packages/master/' + macosversion + '/', 
+        filename: 'p7zip-9.20.1-1.pkg', */
```

Comment:
Commenting out the filename for "darwin" results in line 47 failing when you try to path.join(cwd, undefined)

Reply:
code is commented out since mac binaries are included prepack in repo and npm pulls, only sfx modules downloaded at install time. i removed comment out, refactored since undefined will be returned still.

Reply:
Just to confirm, i did get install time error too. I have updated the code and test install on mac system with sierra with out error. however need i to track down why sfx modules not copied into binary folder also.

i used my forked `npm install node-7z-forall`

Reply:
All issues for install on macos has been resolved, with latest code push.